Makefile

```diff
@@ -1,7 +1,8 @@
 .DEFAULT_GOAL := jobs
 
 jobs-definitions:
-	go run development/tools/cmd/rendertemplates/main.go --config templates/config.yaml
+	go get github.com/kyma-project/test-infra/development/tools/cmd/rendertemplates 
```

Comment:
in the script we have a message that says:
```
go run cmd/rendertemplates/main.go --config ../../templates/config.yaml
    in the development/tools
```
maybe we should add the `cd` command here?

Reply:
cd where?
make should be relative to the current folder? this command works if you are in the folder that contains the make file?

Reply:
yes, make file is in the root directory
I think if we do ```cd development/tools``` it will work